dashboard/src/shared/App.test.ts

```diff
@@ -37,34 +37,6 @@ describe("App", () => {
     jest.resetAllMocks();
   });
 
-  describe("listApps", () => {
```

Comment:
The functionality that this was testing (different endpoint if namespace was present) is no longer relevant at all (since the apis server just receives it either way at the same endpoint now).